src/D2L.CodeStyle.Analyzers/ApiUsage/ServiceLocator/SingletonLocatorAnalyzer.cs

```diff
@@ -128,9 +128,7 @@ Compilation compilation
 		private static ImmutableDictionary<INamedTypeSymbol, int> GetContainerTypes( Compilation compilation ) {
 			var containerTypeMappings = new (string typeName, int containedTypeIdx)[] {
 				new ( "D2L.LP.Extensibility.Activation.Domain.IPlugins`1", 0 ),
-				new ( "D2L.LP.Extensibility.Activation.Domain.IPlugins`2", 1 ),
-				new ( "D2L.LP.Extensibility.Plugins.IInstancePlugins`1",   0 ),
-				new ( "D2L.LP.Extensibility.Plugins.IInstancePlugins`2",   0 )
+				new ( "D2L.LP.Extensibility.Activation.Domain.IPlugins`2", 1 )

```

Comment:
nit: I forget, does C# allow for trailing commas in this situation that would make this diff that little bit clearer?
```suggestion
				new ( "D2L.LP.Extensibility.Activation.Domain.IPlugins`2", 1 ),
```